### ISSUE TITLE
Vmware connection issue after install of NCPA 3.1.1 

### ISSUE THREAD

**Issue by HCR3333** [NagiosEnterprises/ncpa#1218]
We encountered vdi issues installing on our VMware servers.  We could not access VDI from these servers until 3.1.1 was uninstalled (back to 3.1.0.  Has anyone encountered anything similar?  
I will edit and add error messages as soon as I get the exact details.

**Comment by ne-bbahn** [comment]
I can't see anything among the changes to NCPA that should be making a difference here, but NCPA 3.1.1 does update the Python and OpenSSL version, which shouldn't affect your VCS server connections, but it is possible that it's related. Have you tried re-installing NCPA 3.1.1 to ensure that NCPA 3.1.1 is indeed the issue?

**Comment by HCR3333** [comment]
Yes I have tried reinstalling after fixing the issue and still had the same problem.  Its a weird one for sure.

**Comment by ne-bbahn** [comment]
The only change that would make any sense as far as breaking things would be the update to the Python version from 3.11 to 3.12.6 and the update to OpenSSL 3.0.15. I don't see any other way this could break your connection.

**Comment by ne-cdietsch** [comment]
I believe this should no longer be an issue on newer versions of NCPA. I have not been able to reproduce this issue or seen/heard of any similar VMware issues since. 

If it comes up again, we can always re-open the issue.
